lpc55_sign_bin/src/main.rs

```diff
@@ -92,6 +98,12 @@ enum Command {
 
         #[clap(long, default_value_t = 0)]
         image_key_revoke: u16,
+
+        /// TOML file containing field configuration of debug access
+        /// permissions. These may only further restrict the permissions set in
+        /// CMPA.
+        #[clap(long)]
+        debug_settings_cfg: PathBuf,
```

Comment:
(same question -- having to keep this file around any time I want to run these commands is going to be frustrating)